Jenkinsfile

```diff
@@ -0,0 +1,9 @@
+#!groovy
+
+// Don't test plugin compatibility to other Jenkins versions
+// Allow failing tests to retry execution
+// buildPlugin(failFast: false)
+
+// Test plugin compatbility to latest Jenkins LTS
```

Comment:
If students in a workshop would be looking at this `Jenkinsfile`, then I suggest adding links in the comments to:

* https://github.com/jenkins-infra/pipeline-library
* https://jenkins.io/doc/book/pipeline/shared-libraries